qa/suites/rados/basic/tasks/rgw_snaps.yaml

```diff
@@ -26,16 +26,7 @@ tasks:
     - default.rgw.control
     - default.rgw.meta
     - default.rgw.log
-- s3readwrite:
+- s3tests:
```

Comment:
does s3-tests offer (almost) the same coverage as s3tests-test-readwrite ?

Reply:
For the sake of what's needed in the rados suite, yes